web/apps/portal/.env.example

```
@@ -0,0 +1,7 @@
+# Unkey API (defaults to https://api.unkey.dev if not set)
+UNKEY_API_URL=http://localhost:7070
+
+# Database (read-only access for portal config + session lookups)
+DATABASE_HOST=localhost:3306
+DATABASE_USERNAME=unkey
+DATABASE_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_USERNAME key

(UnorderedKey)